### ISSUE TITLE
Two problems encountered in making MODs

### ISSUE THREAD

**Issue by qingqingzixin** [SpectrumQT/WWMI-TOOLS#1]
First of all, thank you very much for your code. However, during the process of creating the MOD, I encountered two issues. The first was that when the number of vertices exceeded 65000, the character's face would disappear or become pulled. The second issue was that the MOD I created seemed to have lost its outline

**Comment by SpectrumQT** [comment]
Hello! Thank you for the report! Investigation was successful and 65k Vertex Limit issue was fixed in [0.8.1](https://github.com/SpectrumQT/WWMI-TOOLS/releases/tag/v0.8.1) update.
Outlines are controlled by multiple Vertex Colors. My only advice is to try painting COLOR1 black and see if it helps. Default models re-imported back in game with WWMI doesn't have any issues with outlines afaik.

Here's the latest info I have regarding channels. It may be inaccurate.

```
COLOR:
R - Outline Mask (prevents outlines on certain areas from being drawn on certain angle)
might be changing colors according to existing Material Functions in-game, needs more testing
G - Outline Thickness (Thanks, Zlevir)
B - Skin Mask (Determines whenever the certain area is skin or not, affects colors and may be more)
A - Outline Thickness (Only affects hair so far, Material Dependant)

Extra Info:
Red values
0 means no draw, 1 means draw

Green Value
Green 0 means no outline, 1 means thicc outlines
However, Alpha at 0 is the exact opposite of Green value.

Blue Value
0 = not skin 
1 = skin

COLOR1:
No info, but painting black seems to fix outlines.
```

**Comment by qingqingzixin** [comment]
Thank you very much for your answer
